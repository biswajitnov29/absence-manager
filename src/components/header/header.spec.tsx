import React from "react";
import { render, screen } from "@testing-library/react";
import SiteHeader from './header';

describe('<SiteHeader />', () => {
    test('renders correctly', () => {
        const { container } = render(<SiteHeader />);
        expect(container).toMatchSnapshot();
    });
});