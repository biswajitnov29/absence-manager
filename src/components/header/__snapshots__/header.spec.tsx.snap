// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SiteHeader /> renders correctly 1`] = `
<div>
  <header
    class="site-header "
  >
    <h2
      class="site-header-appname"
    >
      Absence Management
    </h2>
    <ul
      class="site-header-menu"
    >
      <li
        class="site-header-menu-item"
      >
        <a
          class="site-header-menu-item-link"
          href="#"
        >
          Home
        </a>
      </li>
      <li
        class="site-header-menu-item"
      >
        <a
          class="site-header-menu-item-link"
          href="#"
        >
          Absence Report
        </a>
      </li>
      <li
        class="site-header-menu-item"
      >
        <a
          class="site-header-menu-item-link"
          href="#"
        >
          Members
        </a>
      </li>
      <li
        class="site-header-menu-item"
      >
        <a
          class="site-header-menu-item-link"
          href="#"
        >
          Contact Us
        </a>
      </li>
    </ul>
    <ul
      class="site-header-actionmenu"
    >
      <li
        class="site-header-menu-item site-header-search"
      >
        <a
          class="site-header-menu-item-link"
          href="#"
        >
          Search
        </a>
      </li>
      <li
        class="site-header-menu-item "
      >
        <button
          class="site-button site-button-secondary"
        >
          Log In
        </button>
      </li>
    </ul>
  </header>
</div>
`;
